zuora-api/proxied-request.js

```diff
@@ -14,9 +14,8 @@ const isObject = require('lodash/isObject');
  * @return {[Object]}                suitable object for making http requests from nice-request
  */
 const mergeOptions = (requestOptions, extraOptions) => {
-  const result = isObject(extraOptions) ? Object.assign(requestOptions, extraOptions) : requestOptions;
-
-  return result;
+  let result = Object.assign(config.defaultRequestOptions(), requestOptions);
```

Comment:
won't Object.assign modify defaultRequestOptions with the requestOptions you pass in? Might be safer to use Object.assign({}, defaultRequestOptions, requestOptions).